documentation/HEADER_h.rst

```diff
@@ -0,0 +1,4 @@
+@HEADER@.h
+=======================================
```

Comment:
This will almost always generate a warning with Sphinx, because it expects that the underline is of the same length as the title.

I think it's possible to bypass that by converting the file into markdown:

````markdown
# @HEADER@.h

```{doxygenfile} @HEADER@.h
```
````

Reply:
Fine by me. The documentation for RST said the underlines had to be "as least as long as the title", and I didn't know what the Markdown syntax should be if it was even possible.

Reply:
I pushed the change to generate Markdown instead.